### PR TITLE
253 Update Subsquid version

### PR DESCRIPTION
**Pull request by andrii-kl**

### CODE REVIEW

File: local-testnet/docker-compose.squid-archive.yml [hunk 1]
```diff
@@ -1,49 +1,68 @@
-version: "3.5"
-
 services:
   db:
-    image: postgres:15
-    restart: on-failure
+    image: postgres:12  # CockroachDB cluster might be a better fit for production deployment
```
Comment (XY-Wang) [inline local-testnet/docker-compose.squid-archive.yml:3]: We should be using the latest Postgres version, not a lower version

**Review by XY-Wang — CHANGES_REQUESTED** [review]
Did you run the squid archive with the local node and squid processor to test if they work?

File: local-testnet/README.md [hunk 2]
```diff
@@ -22,3 +22,12 @@ After a successful run
 * The UI is served at http://localhost:3000/
 * The testnet WS endpoint listens at ws://localhost:9944 and can be used with [Polkadot/Substrate Portal](https://polkadot.js.org/apps/?rpc=ws%3A%2F%2F127.0.0.1%3A9944#/explorer) and [Substrate Contracts UI](https://contracts-ui.substrate.io/?rpc=ws://127.0.0.1:9944).
 
+
+## Clean docker state
```
Comment (mfornos) [inline local-testnet/README.md:26]: These are common Docker compose operations. Don't think that they should be included in the README.
But if they are, should be comprehensive and command that you can run "up to start" does not run... and the example should cover the clean up of all the running services, i.e. -f <file1> -f <file2>.
So, for a clean up you can create a simple script like https://github.com/web3labs/epirus-substrate/blob/main/local-testnet/run-all.sh but IMO is not needed.

**Review by mfornos — CHANGES_REQUESTED** [review]
Please, check out the pointed out issues.

And update as well the npm dependency of the squid processor in `squid-ink/` to be sure that runs fine with the latest version of the squid archive (since there are changes on how the squid processors request data to the squid archives).

Please, test it out with a local environment (i.e. real services running on your machine).

**Pull request merged**